tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -287,6 +289,23 @@ void ordering_test_cases() {
         std::forward_list<SynthOrdered> b = {10, 20, 40};
         ordered_containers_test(a, a, b);
     }
+    { // slice
+        std::valarray<int> a(20);
+        for (int i = 0; i < 12; ++i) {
+            a[i] = i;
+        }
+
+        const auto a1 = a[std::slice(2, 3, 4)];
+        const auto a2 = a[std::slice(2, 3, 4)];
+        const auto a3 = a[std::slice(3, 3, 4)];
+        const auto a4 = a[std::slice(2, 4, 4)];
+        const auto a5 = a[std::slice(2, 3, 3)];

```

Comment:
No change requested: This is an excellent set of test cases, exercising all possible codepaths in the newly added operator. Thanks! :smile_cat: